test/include/util/storage_test_util.h

```diff
@@ -78,6 +80,27 @@ struct StorageTestUtil {
     return storage::BlockLayout(attr_sizes);
   }
 
+  // Returns a random Schema that is guaranteed to be valid for a single test instance. Multiple Schemas instantiated
+  // with this function may have non-unique oids, but within a Schema, Column oids are unique.
+  template <typename Random>
+  static catalog::Schema RandomSchema(const uint16_t max_cols, Random *const generator) {
```

Comment:
Should this belong in `storage_test_util`, `Schema` being in the namespace `catalog`?

Reply:
Sure